test/db/tools/rz

```diff
@@ -141,6 +141,7 @@ FILE==
 CMDS=!rizin -R ""
 EXPECT_ERR=<<EOF
 EOF
+BROKEN=1
```

Comment:
I wonder why this got broken?

Reply:
I remember because of the extra log line